### PR TITLE
[#113037209] Test speed up

### PR DESCRIPTION
**Pull request by hovsepm**

### CODE REVIEW

**Comment by hovsepm** [comment]
on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/477/

**Comment by hovsepm** [comment]
on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/479/

**Comment by hovsepm** [comment]
on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/480/

**Comment by hovsepm** [comment]
on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/483/

**Comment by hovsepm** [comment]
new on-demand: http://azuresdkci.cloudapp.net/job/powershell-demand/490/

File: src/Common/Commands.ScenarioTests.Common/Common.ps1 [hunk 1]
```diff
@@ -280,7 +280,9 @@ function Wait-Function
 
     do
     {
-        Start-Sleep -s 5
+        if ($env:AZURE_TEST_MODE -eq "Record"){
+          Start-Sleep -s 5
```
Comment (markcowl) [inline src/Common/Commands.ScenarioTests.Common/Common.ps1]: There is a common sleep function that does exactly this:  https://github.com/Azure/azure-powershell/blob/dev/src/Common/Commands.ScenarioTests.Common/Common.ps1#L306

File: src/ServiceManagement/Sql/Commands.SqlDatabase.Test/TestScripts/Database/ImportExportDatabase.ps1 [hunk 2]
```diff
@@ -121,7 +123,9 @@ function GetOperationStatusWithRequestId
     # Test Get IE status with request id, servername, and login credentials
     do
     {
-        Start-Sleep -m 1500
+        if ($env:AZURE_TEST_MODE -eq "Record"){
```
Comment (markcowl) [inline src/ServiceManagement/Sql/Commands.SqlDatabase.Test/TestScripts/Database/ImportExportDatabase.ps1]: same comment

File: src/ServiceManagement/Sql/Commands.SqlDatabase.Test/TestScripts/Database/ImportExportDatabase.ps1 [hunk 3]
```diff
@@ -84,7 +84,9 @@ function GetOperationStatus
     # Test Get IE status with request object
     do
     {
-        Start-Sleep -m 1500
+        if ($env:AZURE_TEST_MODE -eq "Record"){
```
Comment (markcowl) [inline src/ServiceManagement/Sql/Commands.SqlDatabase.Test/TestScripts/Database/ImportExportDatabase.ps1]: same comment

**Comment by hovsepm** [comment]
on-demand 3: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/491/

**Comment by hovsepm** [comment]
3rd on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/493/

**Comment by hovsepm** [comment]
4th on-demand: http://azuresdkci.cloudapp.net/job/powershell-demand/494/

**Comment by hovsepm** [comment]
3rd on-demand: http://azuresdkci.cloudapp.net/job/powershell-demand/495/

**Comment by hovsepm** [comment]
4th on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/496/

**Pull request merged**